packages/foreman/rubygem-rkerberos/rubygem-rkerberos.spec

```diff
@@ -0,0 +1,97 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name rkerberos
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.5
+Release: 1%{?dist}
+Summary: A Ruby interface for the the Kerberos library
+Group: Development/Languages
+License: Artistic 2.0
+URL: http://github.com/domcleal/rkerberos
```

Comment:
```suggestion
URL: https://github.com/domcleal/rkerberos
```

Reply:
this is marked as resolved, but itsn't? :)